src/pages/index.js

```diff
@@ -73,57 +66,195 @@ function handleImageClick(cardData) {
+      // cardAddForm.setLoading(false);
+      cardSection.addItem(card);
+      addCardFormPopup.close();
+      addCardForm.reset();
+    })
+    .catch((error) => {
+      console.error(error);
+    })
+    .finally(() => {
+      console.log("Add card complete");
+      addCardFormPopup.renderLoading(false);
+    });
 }
 
-addNewCardButton.addEventListener("click", () => {
-  addCardFormPopup.open();
+api
+  .getInitialCards()
+  .then((res) => {
+    console.log(res);
+    cardSection.renderItems(res);
+  })
+
+  .catch((err) => alert(err));
+
+api
+  .getUserInfo()
+  .then((res) => {
+    userInfo.setUserInfo(res);
+    userInfo.updateProfileImage(res);
+  })
+  .catch((err) => alert(err));
+
+//Avatar
+const profileImageForm = document.querySelector("#edit-avatar-form");
+const profileFormValidator = new FormValidator(config, profileImageForm);
+profileFormValidator.enableValidation();
+
+function handleImageProfileEditSubmit(data) {
+  newProfileImageModal.renderLoading(true);
+
+  api
+    .setUserAvatar(data.link)
+    .then((res) => {
+      userInfo.updateProfileImage(res);
+      newProfileImageModal.close();
+      //profileImageForm.reset();
+      // newProfileImageModal.setLoading(false);
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      console.log("Avatar edit submit");
+
+      newProfileImageModal.renderLoading(false);
+    });
+}
+
+const profileImageCover = document.querySelector(".profile__edit-image");
+profileImageCover.addEventListener("click", () => {
+  newProfileImageModal.open();
 });
 
-profileEditButton.addEventListener("click", () => {
-  editProfileFormValidator.resetValidation();
-  const userData = userInfo.getUserInfo();
-  console.log(userData);
-  profileEditPopup.setInputValues({
-    name: userData.name,
-    description: userData.job,
+const newProfileImageModal = new PopupWithForm({
+  popupSelector: "#edit-avatar-modal",
+  handleFormSubmit: handleImageProfileEditSubmit,
+});
+newProfileImageModal.setEventListeners();
+
+// confirmation
+
+function handleDeleteCardSubmit(card) {
+  confirmModal.setSubmitAction(() => {
+    api
+      .deleteCard(card._id)
+      .then((res) => {
+        console.log(res);
+        card.remove();
+        confirmModal.close();
+      })
+      .catch((err) => {
+        console.error(err);
+      })
+      .finally(() => {
+        console.log("Delete card complete");
+      });
   });
+  confirmModal.open(card);
+}
 
-  profileEditPopup.open();
+const confirmModal = new PopupWithConfirmation({
+  popupSelector: "#confirmation-modal",
 });
+confirmModal.setEventListeners();
+
+function likeCard(card) {
+  api
+    .likeCard(card._id, card.isLiked)
+    .then((res) => {
+      console.log(res);
+      card.setIsLiked(res.isLiked);
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      console.log("Like card complete");
+    });
+}
+
+function unlikeCard(card) {
```

Comment:
Please, delete `unlikeCard` not to mislead. You don't use it

Reply:
Whenever I try to delete unlikeCard I get errors...

Reply:
> Whenever I try to delete unlikeCard I get errors...

Please, delete `this._unlikeCard = unlikeCard;` from `Card` because you don't use it. That's why there is an error